Mathlib/FieldTheory/RatFunc.lean

```diff
@@ -16,20 +16,23 @@ import Mathlib.RingTheory.Polynomial.Content
 /-!
 # The field of rational functions
 
+> THIS FILE IS SYNCHRONIZED WITH MATHLIB4.
+> Any changes to this file require a corresponding PR to mathlib4.
+
```

Comment:
Let's not add this!

Reply:
Oops!

Reply:
Technically, what it says is correct, though...